.github/workflows/dev-server-cd.yml

```
@@ -1,37 +1,28 @@
-            git pull origin develop
-            cd server
+      - name: Login to GitHub Container Registry
+        run: echo ${{ secrets.GITHUB_TOKEN }} | docker login ghcr.io -u ${{ github.actor }} --password-stdin
 
-            cp ../.env .
-            chmod +x deploy.sh
-            ./deploy.sh
+      - name: Deploy with Docker Compose
+        run: |
+          cd /home/ubuntu/2025-moment/server
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)